### ISSUE TITLE
Debug with breakpoints

### ISSUE THREAD

**Issue by addremove** [webdriverio/webdriverio#427]
Is it possible currently (or any future plans) to be able to use breakpoints ? When a test fails, it's painful to walk through all the steps to replicate the error. 

Example: I expand a tree, toggle a few nodes, test their values. If I see a failure somewhere down the line, I would love to have the ability to pause the test right there and poke in the console to see what's going wrong. 

Another example: 

      client.elements('.some-class', function(error, result){
          result.value.forEach(function(i){
              client.elementIdElements(i.ELEMENT, '.child-class', function(childError, childResult){              
                  ... if (some_condition){
                          BREAK_POINT
                      }
              })
          })
      });

**Comment by christian-bromann** [comment]
@addremove yes! There are plans to support breakpoints. For some browser it will be tricky as they close the session once you open the dev tools but in general it is possible

**Comment by addremove** [comment]
:+1:  That's awesome, can't wait to use it. I wouldn't mind using any mainstream browser that has the ability to set breakpoints with webdriverio.

**Comment by christian-bromann** [comment]
I figured this is easier then I thought so it will be shipped with the next version `2.5.0`. The command is called `debug`, so you can use it like:

```js
client.elements('.some-class', function(error, result){
      result.value.forEach(function(i){
          client.elementIdElements(i.ELEMENT, '.child-class', function(childError, childResult){              
              ... if (some_condition){
                      client.debug();
                  }
          })
      })
  });
```

Thanks for this suggestion

**Comment by NiGhTTraX** [comment]
@christian-bromann this isn't on npm, can you please publish the latest version?
